### DISCUSSION TITLE
Changing bot's online status?

### DISCUSSION THREAD

**Discussion by Ben569483** [Araan-Sheikh/Clomel-utility#8] [Q&A] [answered]
Where could I change the bot's online status? For example, the bot says that it's status is online but on a phone, how do I make it just online?

**Comment by Araan-Sheikh** [comment]
> Where could I change the bot's online status? For example, the bot says that it's status is online but on a phone, how do I make it just online?

Can you give me ur discord username so tht we could connect each other and you will get your reply soon on discord

**Comment by Ben569483** [comment, marked as answer]
Nevermind I figured it out haha, only just seen it as well.

Just had to change this line, replacing 'Discord Android' with 'discord.js'
`Constants.DefaultOptions.ws.properties.$browser = 'Discord Android'`
